servers/src/main/java/tachyon/master/block/meta/MasterWorkerInfo.java

```diff
@@ -68,8 +72,9 @@ public MasterWorkerInfo(long id, NetAddress address) {
     mStartTimeMs = System.currentTimeMillis();
     mLastUpdatedTimeMs = System.currentTimeMillis();
     mIsRegistered = false;
-    mTotalBytesOnTiers = Collections.emptyList();
-    mUsedBytesOnTiers = Collections.emptyList();
+    mStorageTierAssoc = new StorageTierAssoc(new ArrayList<String>());
+    mTotalBytesOnTiers = new HashMap<String, Long>();
+    mUsedBytesOnTiers = new HashMap<String, Long>();
```

Comment:
Do these need to be initialized to empty collections? Is there a chance we will use it before `register` is called?